graphql_compiler/tests/conftest.py

```diff
@@ -35,6 +50,24 @@ def init_graph():
 
 
 @pytest.fixture(scope='class')
-def graph_client(request, init_graph):
+def graph_client(request, init_snapshot_graph_client):
     """Get a client for an initialized db, with all test data imported."""
-    request.cls.graph_client = init_graph
+    request.cls.graph_client = init_snapshot_graph_client
+
+
+@pytest.fixture(scope='class')
+def integration_graph_client(request, init_integration_graph_client):
+    """Get a client for an initialized db, with all test data imported."""
+    request.cls.graph_client = init_integration_graph_client
+
+
+@pytest.fixture(scope="class")
```

Comment:
nit: use single-quotes around `class` for consistency?

Reply:
Sure will fix.